### PR TITLE
Mouse Event handlers and CSS styled components added

### PR DESCRIPTION
**Pull request by edenlhn**
@FEC-Kickstand/kickstand
Anything after line 17 for the MiniPledgeWidget component and line 65 for the RewardTier component can be safely ignored as they are just my CSS styled components.

Test related & config files can be ignored for now as well; I seem to be having some issue with rebasing with the master branch.

### CODE REVIEW

File: client/components/RewardTier.jsx [hunk 1]
```diff
@@ -1,22 +1,134 @@
...
+          <TitleWrapper className='pledgeAmount'>Pledge US$ {this.props.reward.pledgeAmount} or more</TitleWrapper>
+          <RewardName className='rewardName'>{this.props.reward.name}</RewardName>
+          <RewardDesc className='rewardDesc'>{this.props.reward.description}</RewardDesc>
+          <div className='rewardItems'>
+            <GenericWrapper>INCLUDES:</GenericWrapper>
+            <ul>
+              <ListWrapper>{this.props.reward.item1}</ListWrapper>
+              <ListWrapper>{this.props.reward.item2}</ListWrapper>
+              <ListWrapper>{this.props.reward.item3}</ListWrapper>
+            </ul>
```
Comment (KStockley) [inline client/components/RewardTier.jsx:46]: This seems to indicate that you are hard-coding three reward items into each mock campaign. Is that the case?

File: package.json [hunk 2]
```diff
@@ -5,41 +5,39 @@
...
     "@babel/preset-react": "^7.0.0",
-    "babel-core": "^7.0.0-bridge.0",
     "babel-loader": "^8.0.4",
+    "babel-plugin-styled-components": "^1.8.0",
+<<<<<<< HEAD
     "enzyme": "^3.7.0",
     "enzyme-adapter-react-16": "^1.6.0",
+=======
+>>>>>>> 3e89b6e8121b66daea8c66f6f324e58695761ba8
     "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
```
Comment (KStockley) [inline package.json:38]: Unresolved file conflicts?

File: client/components/MiniPledgeForm.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const MiniPledgeForm = (props) => (
+  <div className='miniPledgeForm'>
+    <LabelWrapper>Pledge amount</LabelWrapper>
+    <div>
+      <CurrencyWrapper for={`${props.reward.id}pledgeForm`}>$</CurrencyWrapper>
+      <InputWrapper type='text' id={`${props.reward.id}pledgeForm`} defaultValue={`${props.reward.pledgeAmount}`} min={`${props.reward.pledgeAmount}`}></InputWrapper>
+    </div>
```
Comment (KStockley) [inline client/components/MiniPledgeForm.jsx:10]: Does eslint complain about not using destructuring when accessing props values on lines 8 and 9?

**Review by KStockley — CHANGES_REQUESTED** [review]
@edenlhn You seem to have an unresolved conflict in your package.json file.
One of your components does not look to be dynamically rendering the rewards on a campaign.
Should you be using destructuring in the MiniPledgeForm?
The mouse behaviour appears to be fine. I cannot really comment on the styling code because I do not know React/component styling yet.

File: client/components/RewardTier.jsx (hunk 1 above)
Comment (edenlhn) [reply client/components/RewardTier.jsx:46]: Yes it is. Every reward tier has three reward items that it needs to display.

File: package.json (hunk 2 above)
Comment (edenlhn) [reply package.json:38]: Yes this is part of a bigger problem I'm having with git right now. Will try to resolve once I can get this PR to merge with my master.

File: client/components/MiniPledgeForm.jsx (hunk 3 above)
Comment (edenlhn) [reply client/components/MiniPledgeForm.jsx:10]: Not on my end, no. I'll try to refactor to use destructuring if that is one of the things that ESlint does not like though.

**Pull request merged**